File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/interfaces/interfaceWithInterfaceInheritance/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.interfaces.interfaceWithInterfaceInheritance.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Fruits = "fruits";
  }

  public static class SEED {
    public static final String TYPE_NAME = "Seed";

    public static final String Name = "name";
  }

  public static class FRUIT {
    public static final String TYPE_NAME = "Fruit";

    public static final String Seeds = "seeds";
  }

  public static class STONEFRUIT {
    public static final String TYPE_NAME = "StoneFruit";

    public static final String Seeds = "seeds";

    public static final String Fuzzy = "fuzzy";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/interfaceWithInterfaceInheritance/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.interfaceWithInterfaceInheritance.expected

import kotlin.String

@Generated
public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  @Generated
  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Fruits: String = "fruits"
  }

  @Generated
  public object SEED {
    public const val TYPE_NAME: String = "Seed"

    public const val Name: String = "name"
  }

  @Generated
  public object FRUIT {
    public const val TYPE_NAME: String = "Fruit"

    public const val Seeds: String = "seeds"
  }

  @Generated
  public object STONEFRUIT {
    public const val TYPE_NAME: String = "StoneFruit"

    public const val Seeds: String = "seeds"

    public const val Fuzzy: String = "fuzzy"
  }
}
